src/group_theory/subgroup/pointwise.lean

```diff
@@ -81,6 +81,10 @@ set_smul_subset_iff
 lemma subset_pointwise_smul_iff {a : α} {S T : subgroup G} : S ≤ a • T ↔ a⁻¹ • S ≤ T :=
 subset_set_smul_iff
 
+/-- Applying a `mul_distrib_mul_action` results in an isomorphic subgroup -/
+def equiv_smul (a : α) (H : subgroup G) : H ≃* (a • H : subgroup G) :=
```

Comment:
Does `@[simps]` generate anything useful? If not, it might be because it's missing from `subgroup_equiv_map` too,

Reply:
I pushed a commit to make `simps` more useful, but you'll still need to add the attribute.